src/plugins/intel_cpu/src/nodes/scaled_attn.cpp

```diff
@@ -1193,39 +1208,84 @@ void ScaledDotProductAttention::createPrimitive() {
         }
     }
     auto rtPrecision = getRuntimePrecision();
+    const auto keyDims = getInputShapeAtPort(1).getDims();
+    const auto valueDims = getInputShapeAtPort(2).getDims();
+    const auto& cpuConfig = context->getConfig();
+    const auto keyS = *(keyDims.end() - 1);
+    const auto valueS = *(valueDims.end() - 1);
 
+    m_key_quant_param.groupSize = cpuConfig.keyCacheGroupSize ? cpuConfig.keyCacheGroupSize : keyS;
+    m_key_quant_param.isByChannel = cpuConfig.keyCacheQuantByChannel;
```

Comment:
As we discussed lets use false as default value for SDPA. Since PA and SDPA are expected to have different default values, we need to use cpuConfig.keyCacheQuantByChannel here only in case it was set explicitly. 
I would propose to reimplement key_cache_quant_bychannel property as enum:
enum class CacheQuantMode {
AUTO,
BY_CHANNEL,
BY_HIDDEN
}
With AUTO being default - to allow operations make their own choice on desired mode

Reply:
Thanks for your advice, I didn't figure it out how to switch between. I will implement it. 
BTW could we just set by_hidden as default in this PR and change to by-channel afther validation team gives enough accuracy data ?

Reply:
>BTW could we just set by_hidden as default in this PR and change to by-channel afther validation team gives enough accuracy data ?

Yeah, I think this is reasonable